mantidproject/mantid
# Vesuvio - update sigma theta value in parameter files

The value for `sigma_theta` in both the `VESUVIO_double_diff_Parametes.xml` and `VESUVIO_single_diff_Parameters.xml` parameter files have been updated to `0.016` as advised by the instrument scientists

**To test:**
- Check `VESUVIO_double_diff_Parametes.xml` and `VESUVIO_single_diff_Parameters.xml` to ensure that the value in both the Forward and Back scattering (forward scattering does not exist in `double_diff`) is now 0.016.
- Ensure all Vesuvio tests pass

Fixes #16804

[RELEASE NOTES](https://github.com/mantidproject/mantid/blob/3de29bb4ca31c80930d977ee3e4d5913fb4d9975/docs/source/release/v3.8.0/indirect_inelastic.rst)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
